src/redundantaudioencoder/RedundantAudioEncoder.ts

```diff
@@ -227,6 +229,21 @@ export default class RedundantAudioEncoder {
     RedundantAudioEncoder.log(`redundancy ${this.redundancyEnabled ? 'enabled' : 'disabled'}`);
   }
 
+  /**
+   * Helper function to only enqueue audio frames if they do not exceed the audio payload byte limit imposed by
+   * Chromium-based browsers. Chromium will throw an error (https://crbug.com/1248479) if an audio payload larger than
+   * 1000 bytes is enqueued. Any controller that attempts to enqueue an audio payload larger than 1000 bytes will
+   * encounter this error and will permanently stop sending or receiving audio.
+   */
+  private maybeEnqueueAudioFrame(
```

Comment:
Since its such a short function we can probably make the name explicit, maybe `enqueueAudioFrameIfPayloadSizeIsValid`? Borderline too verbose, up to you. I myself have felt a bit mid towards the prefix 'maybe' after using it a bit over the years.

Reply:
We can go the verbose route as this is a private function anyway. I only used `maybe` since it looked like an existing pattern but I'm not strongly for that pattern either.